packages/embark-contracts-manager/src/index.js

```diff
@@ -28,11 +28,11 @@ class ContractsManager {
     this.events.setCommandHandler("contracts:build", this.buildContracts.bind(this));
 
     this.events.setCommandHandler('contracts:list', (cb) => {
-      cb(this.compileError, this.listContracts());
+      cb(null, this.listContracts());
```

Comment:
👏

Reply:
Does this mean that we should return this `err` on the compile command and be done with keeping it around?

Reply:
Ah no, that's a rebase mistake on my part